script/git-clone-others.sh

```diff
@@ -113,9 +120,10 @@ for repository in $REPOSITORY_LIST ; do
             cd ..
         fi
 
-        returnCode=$?
-        if [ $returnCode != 0 ] ; then
-            exit $returnCode
+        if [ `echo "$BRANCHED_7_REPOSITORY_LIST" | grep "^$repository$"` ] ; then
+            cd ${repository}
+            git checkout 7.x
```

Comment:
This has a bug if "-b 7.39.x" is given as a script parameter, it will still end up on 7.x instead...
@mbiarnes Good point, I 'll fix that.

Reply:
Fixed